src/main.rs

```diff
@@ -48,24 +48,32 @@ fn main() {
     let args = Args::parse();
     let (nifti_header, data) = read_3d_image::<_, f32>(args.input_image);
 
-    let streamlines = args.fibers.map(|path| {
-        let spacing = Vector3::new(
-            nifti_header.pixdim[1],
-            nifti_header.pixdim[2],
-            nifti_header.pixdim[3],
-        );
-        let mut reader = Reader::new(path).unwrap().to_voxel_space(spacing);
-        reader.streamlines()
-    });
-
+    let streamlines: Vec<Polyline> = match args.fibers {
+        Some(path) => {
+            let spacing = Vector3::new(
+                nifti_header.pixdim[1],
+                nifti_header.pixdim[2],
+                nifti_header.pixdim[3],
+            );
+            let mut reader = Reader::new(path).unwrap().to_voxel_space(spacing);
+            reader
+                .streamlines()
+                .iter()
+                .map(|streamline| streamline.iter().map(|p| vec3(p.x, p.y, p.z)).collect())
+                .collect()
```

Comment:
I understand that this is not actually related to this MR but I have a comment anyway :)

1. You can use `Reader` generator instead of calling `streamlines()`, which allocates and fills a big object.
2. However, it's kinda sad to allocate a vector of vectors (500k allocations for a 500k full brain tractogram) instead of a single allocation like `trk-io`does. I don't actually know how to display polylines with `wgpu`, maybe what you did is necessary, but I wonder if it's possible to keep the streamlines object as-is. nalgebra `Point3` is already, **I think**, a `vec3`, as far as memory is concerned.

Reply:
In the end, it will end up being a single `Vec<FiberVertex>`, so this is not a necessity. I store them in `Vec<Polyline>` for the index generation. I'll look this up.

Reply:
I can try to use `Point3`, but if it's not compatible with `bytemuck`, it might be a pain.

Reply:
In fact, it's better if you keep this change for the next PR :)